src/app/pages/datasets/modules/permissions/components/acl-editor-list/acl-editor-list.component.ts

```diff
@@ -97,6 +97,7 @@ export class AclEditorListComponent implements OnChanges {
   }
 
   onAceSelected(index: number): void {
-    this.store.selectAce(index);
+    this.store.selectAce(null);
```

Comment:
stupid thing, but helps to rerender ix-combobox and other components with correct data!